src/main/resources/elasticsearch/addOnInfoAndVersions-settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Please give some more justification of why you're doing this (ideally in a code comment somewhere)

Reply:
Resolved